samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -1420,6 +1420,52 @@ To onboard devices manually, you can use the `Bulk Onboard Devices <nRF Cloud Po
 
 If you were directed here as part of other instructions, proceed to the next step of those instructions.
 
+Running on native_sim
+=====================
+
+This sample can be run on the `native_sim` target. That way, you can try out connectivity without the need for embedded hardware.
+A Linux host or docker container is required to run the `native_sim` target.
+Some setup is needed in order to connect the application to the network. See the :ref:`networking_with_native_sim` for more information. Here, the `nat.conf` configuration is used.
+Refer to :ref:`nrf_cloud_multi_service_create_device_cred_locally` for creating the necessary device credentials locally.
+Then, modify the `overlay_native_sim.conf` file to use the client id for the credentials you created.
```

Comment:
```suggestion
You can run this sample on the :ref:`zephyr:native_sim` target.
This enables you to try out connectivity without the need for embedded hardware.
A Linux host or docker container is required to run the ``native_sim`` target.
Some setup is needed to connect the application to the network.
See :ref:`zephyr:networking_with_native_sim` for more information.
This sample uses the :file:`nat.conf` configuration.
Refer to :ref:`nrf_cloud_multi_service_create_device_cred_locally` for creating the necessary device credentials locally.
Modify the :file:`overlay_native_sim.conf` file to use the client ID for the credentials you created.
```

Reply:
I agree with Pekka's comments and have no further comments of my own. I'm okay to approve once his questions are answered and suggestions are applied.